sdk/monitor/Azure.Monitor.OpenTelemetry/src/DefaultAzureMonitorOpenTelemetryOptions.cs

```diff
@@ -26,6 +27,13 @@ public void Configure(AzureMonitorOpenTelemetryOptions options)
             {
                 _configuration.GetSection(AzureMonitorOpenTelemetrySectionFromConfig).Bind(options);
             }
+
+            string? connectionString = _configuration?[ConnectionStringEnvironmentVariable];
```

Comment:
Please add test to confirm this works the way intended.

Reply:
We don't have a test framework yet. Planning to merge the test for both exporters and distro. I validated it with the manual test.

Reply:
ok. this is a p1 thing to have sufficient coverage. reading connectionstring from unintended place/priority or not reading it at all are big support costs.